collect_app/src/main/java/org/odk/collect/android/activities/DataManagerList.java

```diff
@@ -195,7 +195,7 @@ private void createDeleteInstancesDialog() {
         mAlertDialog = new AlertDialog.Builder(this).create();
         mAlertDialog.setTitle(getString(R.string.delete_file));
         mAlertDialog.setMessage(getString(R.string.delete_confirm,
-                mSelected.size()));
+                String.valueOf(mSelected.size())));
```

Comment:
Actually, I do remember there being some kind of explanation. Let me try linting before the change and get back to you.